### ISSUE TITLE
roapi-http can not work on windows

### ISSUE THREAD

**Issue by tempbottle** [roapi/roapi#65]
run roapi-http.exe on windows with simple:
```
roapi-http.exe --table 'tresult=d:/data/out_result.csv'
```
and get error:
```
unsupported extension in uri: d:/data/out_result.csv
```

**Comment by tempbottle** [comment]
[https://github.com/roapi/roapi/blob/main/columnq/src/table/mod.rs#L279](https://github.com/roapi/roapi/blob/main/columnq/src/table/mod.rs#L279) here got csv' with more char of '

**Comment by houqp** [comment]
interesting, @tempbottle I don't have a windows machine to test this. Based on google, it looks like windows terminal doesn't support single quote, could you give `roapi-http.exe --table "tresult=d:/data/out_result.csv"` a try?

**Comment by tempbottle** [comment]
```
roapi-http.exe --table "tresult=d:/data/out_result.csv"
```
can make advance. But `d:` is bad for `uriparse`. I need 
```
roapi-http.exe --table "tresult=filesystem://d:/data/out_result.csv"
```
to bypass `uriparse`. And we can add `file://` schema for this by: [https://github.com/tempbottle/roapi/commit/a15749331bcc1763a3ddb6ccb5ff3e28b91f3823](https://github.com/tempbottle/roapi/commit/a15749331bcc1763a3ddb6ccb5ff3e28b91f3823)

But in the end it failed, because it got block somewhere.

**Comment by houqp** [comment]
good catch @tempbottle , I don't have a windows environment to catch all these edge-cases. do you mind sending your file schema change as a PR?

**Comment by tempbottle** [comment]
OK, I'm a little busy these days, I'll give this a try later.

**Comment by ekroon** [comment]
This issue is fixed in #69 with (https://github.com/roapi/roapi/pull/69#issuecomment-912983676)
